tr.gov.uyap.Editor.Template.desktop

```diff
@@ -0,0 +1,14 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+
+Name=UYAP Template Editor
+Name[tr]=UYAP Şablon Editör
+Comment=UYAP Template Editor is an application to display and create USF documents in UYAP system.
+Comment[tr]=Uyap Şablon Editör uygulaması UYAP sisteminde oluşturulmuş USF dokümanlarının görüntülenmesi için hazırlanmıştır.
+Categories=Office;Java;
+MimeType=application/usf;
+
+Icon=tr.gov.uyap.Editor
```

Comment:
shouldn't this be a different icon?

Reply:
Apparently not, it's not a different app per se. Just launches the app in a different mode. The app itself ships with two different desktop files without functionality to change inside the app. In order to not lose any functionality, I followed the same. I can add a saturated version if you'd like

Reply:
if that's upstream UX then so be it.